affine/__init__.py

```diff
@@ -385,7 +385,7 @@ def __mul__(self, other):
                 vx, vy = other
             except Exception:
                 return NotImplemented
-            return (vx * sa + vy * sd + sc, vx * sb + vy * se + sf)
+            return (vx * sa + vy * sb + sc, vx * sd + vy * se + sf)
```

Comment:
This looks like a bug with the correct fix. The 3x3 matrix is [sa, sb, sc; sd, se, sf] and the 3x1 matrix is [vx; vy; 1]. Can anyone else confirm?